xbmc/video/windows/GUIWindowFullScreen.cpp

```diff
@@ -31,6 +31,7 @@
 #include "video/dialogs/GUIDialogAudioSubtitleSettings.h"
 #include "guilib/GUIWindowManager.h"
 #include "input/Key.h"
+#include "pvr/PVRGUIActions.h"
```

Comment:
Same "problem" as with GUIWindowVisualisation.cpp

As I said, I will get rid of the PVR dependency in another PR. IMO, larger changes are required for this.

Compared to the fat PVR Manager, PVR GUI Actions is tiny (very few includes needed e.g.)